tests/test_api_pytest.py

```diff
@@ -0,0 +1,36 @@
+import subprocess
+import tempfile
+
+import pytest
+
+import dbusmock
```

Comment:
```suggestion
import dbusmock
import pytest
import subprocess
import tempfile
```
also I just realised this should probably have a copyright header in it like other files, feel free to copy/paste whatever's in the rest of the repo (and this goes for any of my future changes in this repo too).

Reply:
Thanks, added.